options.js

```diff
@@ -30,8 +30,9 @@ function onOptionsPageSave(e)
 		"ignore_prefpages": document.querySelector("#ignore_prefpages").checked,
 		"ignore_maps": document.querySelector("#ignore_maps").checked,
 		"ignore_flights": document.querySelector("#ignore_flights").checked,
-	        "dont_override_containers": document.querySelector("#dont_override_containers").checked,
-	        "whitelist": validate_whitelist()
+    "dont_override_containers": document.querySelector("#dont_override_containers").checked,
```

Comment:
You should keep the same amount of spaces/tabs, but otherwise fine.

Reply:
you're right, I don't know why Atom parse spaces different 🤦🏻